mod-finance/schemas/order_transaction_summary.json

```diff
@@ -9,7 +9,8 @@
     },
     "numTransactions": {
       "description": "Total number of transactions (encumbrances) expected for this order",
-      "type": "integer"
+      "type": "integer",
+      "minimum": 1
```

Comment:
I'm not sure it's supported by RMB. Could you add unit to test that appropriate error is returned?

Reply:
Right it throws 500, without proper error message. Created https://issues.folio.org/browse/RMB-521.
Going to close this PR for now